source/guide_restic.rst

```diff
@@ -0,0 +1,211 @@
+.. highlight:: console
+.. author:: Hafnium <haf@hafnium.me>
+
+.. tag:: lang-go
+.. tag:: file-storage
+.. tag:: backup
+.. tag:: sync
+.. tag:: bash
+
+.. sidebar:: About
+
+  .. image:: _static/images/restic.png
+      :align: center
+
+#########
+Restic
+#########
+
+.. tag_list::
+
+Restic_ is a fast and secure backup program, which have compatibility with multiple cloud providers.
+
+----
+
+.. note:: For this guide you should be familiar with the basic concepts of
+
+  * :manual:`Go <lang-go>`
+  * :manual:`SSH <basics-ssh>`
+  * :manual:`cronjobs <daemons-cron>`
+
+License
+=============
+
+Restic is an open source software distributed under the BSD-2-Clause License. All relevant legal information can be found here
+
+  * https://github.com/restic/restic/blob/master/LICENSE
+
+
+Prerequisites
+=============
+
+We're using :manual:`Go <lang-golang>` version go1.12.5 to compile the software.
+
+::
+
+ [isabell@stardust ~]$ go version
+ go version go1.12.5 linux/amd64
+ [isabell@stardust ~]$
+
+If you want to back up to a remote server, you should set up a bucket a or user first. Restic currently supports backup with:
+
+  - `Local directory <https://restic.readthedocs.io/en/latest/030_preparing_a_new_repo.html#local>`__
+  - `sftp server (via SSH) <https://restic.readthedocs.io/en/latest/030_preparing_a_new_repo.html#sftp>`__
+  - `HTTP REST server <https://restic.readthedocs.io/en/latest/030_preparing_a_new_repo.html#rest-server>`__ (`protocol <doc/100_references.rst#rest-backend>`__ `rest-server <https://github.com/restic/rest-server>`__)
+  - `AWS S3 <https://restic.readthedocs.io/en/latest/030_preparing_a_new_repo.html#amazon-s3>`__ (either from Amazon or using the `Minio <https://minio.io>`__ server)
+  - `OpenStack Swift <https://restic.readthedocs.io/en/latest/030_preparing_a_new_repo.html#openstack-swift>`__
+  - `BackBlaze B2 <https://restic.readthedocs.io/en/latest/030_preparing_a_new_repo.html#backblaze-b2>`__
+  - `Microsoft Azure Blob Storage <https://restic.readthedocs.io/en/latest/030_preparing_a_new_repo.html#microsoft-azure-blob-storage>`__
+  - `Google Cloud Storage <https://restic.readthedocs.io/en/latest/030_preparing_a_new_repo.html#google-cloud-storage>`__
+  - And many other services via the `rclone <https://rclone.org>`__ `Backend <https://restic.readthedocs.io/en/latest/030_preparing_a_new_repo.html#other-services-via-rclone>`__
+
+Installation
+============
+
+``cd`` to your ``/$HOME`` home directory before fetching Restic.
+
+We're building restic from source with the following steps:
+
+::
+
+ [isabell@stardust ~]$ git clone https://github.com/restic/restic.git
```

Comment:
restic also provides binary releases: https://github.com/restic/restic/releases/tag/v0.9.6 please use those instead of compiling it to save on time and resources. It will also make the guide a bit shorter because we don't need to mention go or the go version at all.

Reply:
Fix in commit 2ff100c